static/js/admin/vue-permissions-editor/package.json

```diff
@@ -15,20 +15,22 @@
     "lodash.clonedeep": "^4.5.0",
     "lodash.findindex": "^4.6.0",
     "lodash.isempty": "^4.4.0",
-    "vue": "^2.6.14",
-    "vue-router": "^3.5.3"
+    "vue": "^3.2.28",
+    "vue-router": "^4.0.12",
+    "vuex": "^4.0.2"
   },
   "devDependencies": {
     "@testing-library/jest-dom": "^5.16.1",
     "@vue/cli-plugin-babel": "^4.5.15",
     "@vue/cli-plugin-unit-jest": "^4.5.15",
     "@vue/cli-service": "^4.5.15",
-    "@vue/test-utils": "1.3.0",
-    "babel-jest": "^27.4.6",
+    "@vue/compiler-sfc": "^3.2.28",
+    "@vue/test-utils": "2.0.0-rc.18",
+    "babel-jest": "^26.6.3",
```

Comment:
hm, so babel-jest got downgraded, and vue-jest is an alpha? Feel a little weird using an alpha, but if that's what we're stuck with...